src/sentry/web/frontend/base.py

```diff
@@ -303,6 +304,14 @@ def dispatch(self, request: Request, *args: Any, **kwargs: Any) -> Response:
         if self.is_sudo_required(request, *args, **kwargs):
             return self.handle_sudo_required(request, *args, **kwargs)
 
+        if (
+            is_using_customer_domain(request)
+            and "organization_slug" in inspect.signature(self.convert_args).parameters
```

Comment:
Do we need to inspect the signature? We could just add the keyword argument. All implementations of `convert_args` already accept `**kwargs`.

Reply:
@markstory I tried this before, and it led to errors with `got an unexpected keyword argument 'organization_slug'`. I tried it in [`166b3f2` (#44277)](https://github.com/getsentry/sentry/pull/44277/commits/166b3f247b7fdb5cf5a2aef82b4664c3c96b5fa1).